src/components/toggle/toggle/toggle.ts

```diff
@@ -67,6 +67,7 @@ export class SbbToggleElement extends LitElement {
   @property({ attribute: 'disable-animation', reflect: true, type: Boolean })
   public disableAnimation = false;
 
+  private _loaded = false;
```

Comment:
would it work to just us hasUpdated of lit https://lit.dev/docs/components/lifecycle/?